emData/generate_PR.py

```diff
@@ -74,7 +74,7 @@ def writeSourceModuleInstance(module, wiresFileName):
     strSource += "#pragma HLS inline off\n"
     strSource += "#pragma HLS interface mode=ap_vld register port=bx_o\n"
     for i in range(TProjCount):
-        strSource += "#pragma HLS interface mode=ap_memory port=projin[" + str(i) + "].get_mem() latency=2\n"
```

Comment:
if I am reading this diff correctly, there is no longer any pragma for these interfaces; I think we do need that?

Reply:
also it looks like with this change the loop is empty, so if the line should be removed the `for` statement can also be removed.